test/bats.bats

```diff
@@ -37,7 +37,7 @@ fixtures bats
   run bats "$FIXTURE_ROOT/passing.bats"
   [ $status -eq 0 ]
   [ ${lines[0]} = "1..1" ]
```

Comment:
No problem. Just that it' was not "part" of the modification I made. I'm modifying it. no more pingpong about it. :+1: